docs/tutorial_tripole_grid.ipynb

```diff
@@ -996,22 +996,215 @@
    "metadata": {},
```

Comment:
<p>I suggest: "using&nbsp;the <code>REGULAR_WITH_LAND</code>&nbsp;and&nbsp;<code>TRIPOLAR_REGULAR_WITH_LAND</code>&nbsp;Laplacians"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NoraLoose/gcm-filters/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>